frontend-react/src/hooks/network/ServiceHooks.ts

```diff
@@ -0,0 +1,109 @@
+import {
+    Dispatch,
+    SetStateAction,
+    useCallback,
+    useEffect,
+    useMemo,
+    useState,
+} from "react";
+import { AccessToken } from "@okta/okta-auth-js";
+import _ from "lodash";
```

Comment:
Let's only import what we need from lodash:

```js
import pick from "lodash.pick"
```

Historically, lodash has been bad with tree-shaking so `import _ from "lodash"` would load in the entire library into the bundle.  `import specificThing from lodash.specificThing` makes it so we only include what we need in the bundle.  (Not sure if this is still the case, but it couldn't hurt -- especially since we don't have the larger `lodash` dependency listed in package.json.)